Update distributed_inference.md to include a fuller example on distributed inference (#9152)
huggingface/diffusers: docs/source/en/training/distributed_inference.md

<Tip>

> [Here](https://gist.github.com/sayakpaul/cfaebd221820d7b43fae638b4dfa01ba) is a more full-fledged example of running inference across multiple GPUs. To learn more, take a look at the [Distributed Inference with 🤗 Accelerate](https://huggingface.co/docs/accelerate/en/usage_guides/distributed_inference#distributed-inference-with-accelerate) guide.

Reviewer: ```suggestion
Refer to this minimal example [script](https://gist.github.com/sayakpaul/cfaebd221820d7b43fae638b4dfa01ba) for running inference across multiple GPUs. To learn more, take a look at the [Distributed Inference with 🤗 Accelerate](https://huggingface.co/docs/accelerate/en/usage_guides/distributed_inference#distributed-inference-with-accelerate) guide.
```
